list/list.js

```diff
@@ -1511,6 +1511,9 @@ canReflect.assignSymbols(DefineList.prototype,{
 	// get/set
 	"can.getKeyValue": DefineList.prototype.get,
 	"can.setKeyValue": DefineList.prototype.set,
+	"can.getName": function() {
+		return this.constructor.name + "[" + this._cid + "]";
```

Comment:
I think it's better to use `CID(this)` here, from can-cid, in case the implementation changes.

Reply:
makes sense, will do.